src/main/webapp/app/exam/participate/information/exam-information.component.ts

```diff
@@ -24,7 +24,30 @@ export class ExamInformationComponent {
         return this.exam.endDate;
     }
 
-    examOverMultipleDays() {
+    normalWorkingTime(): number | undefined {
```

Comment:
I think it would be better to move these into util files or in some kind of exam date service so that they can be reused in different views / components. Please also add some JavaDoc comments